pages/api/v1/auth/actions/reportContent.ts

```diff
@@ -83,13 +84,26 @@ async function handler(req: NextApiRequest, res: NextApiResponse<IReportContentR
 		const totalUsers = data.count || 0;
 
 		if(type == 'post' && checkReportThreshold(totalUsers) ){
-			_sendPostSpamReportMail(network,strPostType,contentId,totalUsers);
+			const postReportKey = await redisGet(`postReportKey-${post_id}`);
```

Comment:
post_id can be same for networks and even post types

please use `postReportKey-${network}_${strPostType}_${post_id}`

Reply:
resolved